coral-presto/src/test/java/com/linkedin/coral/presto/rel2presto/HiveToPrestoConverterTest.java

```diff
@@ -5,26 +5,38 @@
  */
 package com.linkedin.coral.presto.rel2presto;
 
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 import org.apache.calcite.rel.RelNode;
-import org.apache.hadoop.hive.metastore.api.MetaException;
-import org.apache.hadoop.hive.ql.metadata.HiveException;
+import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeTest;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
+import static com.google.common.io.MoreFiles.deleteRecursively;
+import static com.google.common.io.RecursiveDeleteOption.ALLOW_INSECURE;
 import static org.testng.Assert.*;
 
 
 public class HiveToPrestoConverterTest {
 
+  Path metastoreDbDirectory;
   RelToPrestoConverter relToPrestoConverter;
 
   @BeforeTest
-  public void beforeClass() throws HiveException, MetaException {
-    TestUtils.initializeViews();
+  public void beforeClass() throws Exception {
+    metastoreDbDirectory = Files.createTempFile("coral-presto", "metastore.db");
+    Files.delete(metastoreDbDirectory); // it will be re-created
```

Comment:
What happens if we skip this line? We can delete it unless it throws an exception.